lib/lz4.h

```diff
@@ -50,8 +50,10 @@ extern "C" {
 #define LZ4_VERSION_MAJOR    1    /* for breaking interface changes  */
 #define LZ4_VERSION_MINOR    7    /* for new (non-breaking) interface capabilities */
 #define LZ4_VERSION_RELEASE  2    /* for tweaks, bug-fixes, or development */
+#define LZ4_VERSION_STRING   "1.7.2"
```

Comment:
In order to avoid accidental unsynchronization, a personal favorite would be for `LZ4_VERSION_STRING` to be generated from `LZ4_VERSION_MAJOR`, `LZ4_VERSION_MINOR` and `LZ4_VERSION_RELEASE`.

There is an equivalent construction available in zstd cli (might be shortened a bit) :
https://github.com/Cyan4973/zstd/blob/master/programs/zstdcli.c#L76